src/main/java/com/zaxxer/hikari/pool/PoolBagEntry.java

```diff
@@ -182,8 +182,10 @@ public boolean compareAndSet(int expect, int update)
    @Override
    public String toString()
    {
-      return String.format("%s (created %s, last release %dms ago, %s)",
-                           connection, formatDateTime(creationTime), ClockSource.INSTANCE.elapsedMillis(lastAccess), stateToString());
+      return connection
+         + ", created " + formatDateTime(creationTime)
+         + ", last release " + ClockSource.INSTANCE.elapsedMillis(lastAccess) + "ms ago, "
+         + stateToString();
```

Comment:
effect would be same but less clutter without 'appoend' and who knows '+' be optimized more in future?